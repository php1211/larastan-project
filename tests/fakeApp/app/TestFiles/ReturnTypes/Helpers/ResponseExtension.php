<?php

declare(strict_types=1);

namespace App\TestFiles\ReturnTypes\Helpers;

use Illuminate\View\View;
use Illuminate\Http\Response;
use Illuminate\Contracts\Routing\ResponseFactory;

class ResponseExtension
{
    public function testSimpleView(): Response
    {
        return response('test');
    }

    public function testFactory(): ResponseFactory
    {
        return response();
    }

    public function testFail(): View
    {
        return response('test');
    }
}
