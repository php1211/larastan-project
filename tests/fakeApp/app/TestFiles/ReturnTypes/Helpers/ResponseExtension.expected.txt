23:Method App\TestFiles\ReturnTypes\Helpers\ResponseExtension::testFail() should return Illuminate\View\View but returns Illuminate\Http\Response.
